spec/findbugs_translate_checkstyle_format/translate_spec.rb

```diff
@@ -40,6 +40,35 @@
         expect(doc.get_elements('/checkstyle/file').first.attribute('name').value).to eq('test.java')
       end
     end
+    context 'single BugInstance and many SourceLine' do
+      xml = {
+        'BugCollection' => {
+          'BugInstance' => [
+            {
+              'SourceLine' => {
+                '@classname' => 'com.example.Test',
+                '@start' => 12
+              },
+            },
+            {
+              'SourceLine' => {
+                '@classname' => 'com.example.Test',
+                '@start' => 14
+              },
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)